### PR TITLE
[DP][DDCE-1900] Bootstrap v5.3 update

### PR DESCRIPTION
**Pull request by DannyPennington97**
# DDCE-1900

There were a whole bunch of models that were being weird after adding the play-json-joda dependency to fix compiler warnings. I believe it's to do with the inheritance from the ErrorResponse and the implicit reads/writes that's going on - swapping them all to use the `asJson` method instead of `Json.toJson` fixed it so there we go

Also updated minor dependencies

## Checklist PR Raiser
- [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
- [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
- [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [x]  I've squashed my commits - including the JIRA issue number in the commit message
- [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/uk/gov/hmrc/lisaapi/controllers/Responses.scala [hunk 1]
```diff
@@ -96,8 +96,6 @@ case object ErrorApiNotAvailable extends ErrorResponse(403, "API_NOT_AVAILABLE",
 
 case object ErrorUnauthorized extends ErrorResponse(401, "UNAUTHORIZED", "Bearer token is missing or not authorized")
 
-case object ErrorNotFound extends ErrorResponse(404, "NOT_FOUND", "Resource was not found")
```
Comment (LetMeTestThat) [inline app/uk/gov/hmrc/lisaapi/controllers/Responses.scala:99]: I'm guessing this was just a case of these objects not being used?

Comment (DannyPennington97) [reply app/uk/gov/hmrc/lisaapi/controllers/Responses.scala:99]: Correctamundo

**Comment by LetMeTestThat** [comment]
I noticed you changed some unecessary pattern match syntax to anon functions. There's still two places this can be refactored for completion, if you'd like: GetLisaAccountResponse:85 and ReportWithdrawalChargeRequest:359.

Other than that, ready to merge!

**Comment by DannyPennington97** [comment]
Good spot, I'll add that in and push it up

**Comment by DannyPennington97** [comment]
Ok, changes pushed and squashed!

**Review by LetMeTestThat — APPROVED** [review]

**Pull request merged**